src/anyio/abc/_eventloop.py

```diff
@@ -46,6 +50,7 @@
 
 T_Retval = TypeVar("T_Retval")
 PosArgsT = TypeVarTuple("PosArgsT")
+StrOrBytesPath: TypeAlias = Union[str, bytes, "PathLike[str]"]
```

Comment:
The annotations in Trio's `open_process()` indicates that it doesn't accept `bytes` as the command.

Reply:
That actually means that I need to drop `bytes` from the union too.

Reply:
I did the opposite and added full support for both on all back-ends, fsdecoding to `str` on trio since it doesn't support `bytes` arguments.

Reply:
> nit: In case you want to remove the duplicate `StrOrBytesPath` definitions,
> 
> ```python
> if TYPE_CHECKING:
>     from _typeshed import StrOrBytesPath
> ```
> 
> is documented as stable.

Changed.

Reply:
Reverted, as the docs build fails because it can't find the type at run time.